### ISSUE TITLE
Set `channels_first=False` in MARS classifier

### ISSUE THREAD

**Issue by dxoigmn** [twosixlabs/armory#1036]
In ART, the default value of `channels_first` in `PyTorchClassifier` is `True`. However, the baseline MARS model specifies the input shape to have the channels axis as the last axis:
https://github.com/twosixlabs/armory/blob/2b72bea09cdcf6e1d4d812d1a4bc68d96d2cd665/armory/baseline_models/pytorch/ucf101_mars.py#L310

This is really only a problem when one tries to apply attacks in ART that work with estimators that require `channels_first=False`.

I haven't check any other classifiers in Armory; it may be worth review this default value of `channels_first`.

**Comment by lcadalzo** [comment]
Thanks @dxoigmn for raising this issue.

The following ART 1.6.0 attacks appear to use `channels_first` attribute:

adversarial_patch_*.py
		dpatch
		dpatch_robust
		pixel_threshold
		shadow_attack
		spatial_transformation
		square_attack
		wasserstein
		zoo
		poisoning/perturbations/image_perturbations.py
		
The default value for Keras/TF classifiers in ART is `channels_first=False`, which aligns with the TF/Keras Armory baseline models. So this issue should only be affecting Armory's PyTorch baseline model(s)

**Comment by lcadalzo** [comment]
Closing this issue since it's a duplicate of #988
